Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM debian:buster
+
+MAINTAINER Sandro Jäckel <sandro.jaeckel@gmail.com>
+
+WORKDIR /app
+
+COPY ["docker/config.json", "/root/.config/Red-DiscordBot/"]
```

Comment:
You either didn't properly setup the file needed, or (more likely) this got excluded because you also did not update `.gitignore` to allow this file to be added automatically